db/migrate/20210715154620_alter_posts_authors_locked_default.rb

```diff
@@ -0,0 +1,5 @@
+class AlterPostsAuthorsLockedDefault < ActiveRecord::Migration[5.2]
+  def change
+    change_column_default :posts, :authors_locked, from: false, to: true
```

Comment:
(Thought, discussed on Discord a bit - might be a smaller changeset if we change the default in the controller rather than the model?)